ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -8304,3 +8308,19 @@ DTPM:
   - detergent_pump_run_command
   implements:
   - MONITORING
+
+APM:
+  guid: "db162c18-9976-4e80-9aa4-6dd61d0c6970"
+  description: "Atmospheric pressure monitoring"
+  uses:
+  - air_pressure_sensor
+  implements:
+  - MONITORING
+
+WVDM:
```

Comment:
please change to "ZWVDM" to help indicate that this is for the zone

Reply:
updated thank you